src/components/ProgramRecordSendModal/SendLearnerRecordModal.jsx

```diff
@@ -0,0 +1,155 @@
+import {
+  ModalDialog, Form, SelectableBox, ActionRow, Button,
+} from '@edx/paragon';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+
+import sendRecords from './data/service';
+
+const SendLearnerRecordModal = ({
+  isOpen, toggleSendRecordModal, creditPathways, uuid, username, setSendRecord,
+}) => {
+  const [selectedPathways, setSelectedPathways] = useState([]);
+
+  const handleSendRecords = async () => {
+    const pathwaysToSendRecords = creditPathways.filter(pathway => selectedPathways.includes(pathway.name));
+
+    const orgs = {
+      sendRecordSuccessOrgs: [],
+      sendRecordFailureOrgs: [],
+    };
+
+    await Promise.all(pathwaysToSendRecords.map(pathway => sendRecords(uuid, username, pathway.id)
+      .then(response => {
+        if (response.status >= 200 && response.status < 300) {
+          orgs.sendRecordSuccessOrgs.push(pathway);
+        } else {
+          orgs.sendRecordFailureOrgs.push(pathway);
+        }
+      })
+      .catch(error => {
+        orgs.sendRecordFailureOrgs.push(pathway);
+        const errorMessage = (`Error: Could not send ${pathway.name} record: ${error.message}`);
+        logError(errorMessage);
+      })));
+
+    setSendRecord(prev => ({
+      ...prev,
+      ...orgs,
+    }));
+
+    toggleSendRecordModal();
+  };
+
+  const handleCheckboxChange = (e) => {
+    e.persist();
+
+    if (e.target.checked) {
+      setSelectedPathways(prev => ([
+        ...prev,
+        e.target.value,
+      ]));
+    } else {
+      setSelectedPathways(prev => ([
+        ...prev.filter(pathway => pathway !== e.target.value),
+      ]));
+    }
+  };
+
+  return (
+    <>
+      <ModalDialog
+        title="Send Program Record"
+        isOpen={isOpen}
+        hasCloseButton
+        onClose={toggleSendRecordModal}
+        size="lg"
+        className="send-record-modal"
+        isFullscreenScroll
+      >
+        <ModalDialog.Header>
+          <ModalDialog.Title>
+            <FormattedMessage
+              id="send.record.modal.header"
+              defaultMessage="Send Program Record to edX Credit Partner"
+              description="Heading for the send record modal dialog box"
+            />
+          </ModalDialog.Title>
+        </ModalDialog.Header>
+        <ModalDialog.Body>
+          <p>
+            <FormattedMessage
+              id="send.record.modal.description"
+              defaultMessage="You can directly share your program record with edX partners that accept credit for this Professional Certificate Program. Once you send your record you cannot unsend it."
+              description="Information about what the send program record modal is used for"
+            />
+          </p>
+          <Form>
+            <p>
+              <FormattedMessage
+                id="send.record.modal.form.header"
+                defaultMessage="Select organization(s) you wish to send this record to:"
```

Comment:
Nit: I feel like this is missing a "the" somewhere? `Select <the> organization(s) you wish to send this record to:`?